.github/workflows/deploy.yml

```
@@ -0,0 +1,105 @@
+    branches: [ main ]
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  test:
+    uses: ./.github/workflows/backend-tests.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/backend-tests.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)